tests/baselines/reference/enumTagCircularReference.errors.txt

```diff
@@ -3,7 +3,7 @@ tests/cases/conformance/jsdoc/bug27142.js(1,5): error TS2456: Type alias 'E' cir
 
 ==== tests/cases/conformance/jsdoc/bug27142.js (1 errors) ====
     /** @enum {E} */
-        ~~~~~~~~~
+        ~~~~~~~~~~

```

Comment:
Not ideal

Reply:
No, but this is an existing bug made slightly worse here. #41974